### PR TITLE
[#9252] Javadoc: Fix Inspection Results report by IntelliJ 

### LINKED ISSUE #9252: Javadoc: Fix Inspection Results report by IntelliJ
image

image

### PR DESCRIPTION
**Pull request by nkshschdv**
Fixes #9252 

There are still some 6 errors left , but might need help in resolving that see below 
test

### CODE REVIEW

File: src/main/java/teammates/common/util/Const.java [hunk 1]
```diff
@@ -876,6 +876,9 @@ private Const() {
         public static final String STUDENT_FEEDBACK_RESULTS_PAGE = "/page/studentFeedbackResultsPage";
         public static final String INSTRUCTOR_FEEDBACK_SUBMISSION_EDIT_PAGE = "/page/instructorFeedbackSubmissionEditPage";
         public static final String INSTRUCTOR_FEEDBACK_RESULTS_PAGE = "/page/instructorFeedbackResultsPage";
+        public static final String INSTRUCTOR_EDIT_STUDENT_FEEDBACK_PAGE = "/page/instructorEditStudentFeedbackPage";
```
Comment (xpdavid) [inline src/main/java/teammates/common/util/Const.java]: We are in migration process so these `*PAGE` references are supposed to be deleted. You should not added it back. For places where it reference such variable. You should leave them there.

Comment (nkshschdv) [reply src/main/java/teammates/common/util/Const.java]: so that means we need to leave the error as it is ???

Comment (xpdavid) [reply src/main/java/teammates/common/util/Const.java]: Yes. As the file will be gradually migrated. There is no point to update them.

**Comment by nkshschdv** [comment]
Hi @xpdavid , could you please help me in other remaining six errors ? 
I assume i need to use @see instead of @link as these variables and function are inaccessible due to access specifier constraints ??

**Comment by nkshschdv** [comment]
Hi @xpdavid , I pushed the changes. Thanks

File: src/test/java/teammates/test/cases/browsertests/AdminAccountDetailsPageUiTest.java [hunk 2]
```diff
@@ -11,7 +11,7 @@
 import teammates.test.pageobjects.AdminAccountDetailsPage;
 
 /**
- * SUT: {@link Const.ActionURIs#ADMIN_ACCOUNT_DETAILS_PAGE}.
+ * SUT: {@link Const.WebPageURIs#ADMIN_ACCOUNTS_PAGE}.
```
Comment (xpdavid) [inline src/test/java/teammates/test/cases/browsertests/AdminAccountDetailsPageUiTest.java]: I don't think you understanding the full context of the migration process yet. Therefore, my suggestion is that leave all `*UiTest`'s `SUT` documentation there and don't touch them.

**Comment by xpdavid** [comment]
Closed due to inactivity.

**Pull request closed without merging**